### DISCUSSION TITLE
quick question about BANK()

### DISCUSSION THREAD

**Discussion by starleaf-luna** [gbdev/rgbds#1261] [Discussions]
so BANK(x) returns the bank of x, correct? what if MBC5 is used and x is in bank $100-$1ff, and the code used is `ld a,BANK(x)` or loading to another bank? I expect `a` will contain the lowest 8 bits of the bank no., but where will the 9th bit be stored? (in carry?)

edit: tried it; `Value 0x1fe is not 8-bit`...

**Comment by aaaaaa123456789** [comment]
`BANK(xxx)` is returning the value, which is $1FE in your case. There is nothing special about that.
The rest is the error you'd normally get for trying to write that instruction: `ld a, $1fe` directly will give you the same error. If you actually do this, the value will be truncated (it's equivalent to `ld a, $fe`) and it won't do what you want.
